heron/api/src/java/BUILD

```diff
@@ -45,3 +45,16 @@ java_library(
     srcs = glob(["com/twitter/heron/classification/**/*.java"]),
     resources = ["//heron/api/src:processor-service-conf"],
 )
+
```

Comment:
generate_pom.cc should go into tools - this looks a generation tool that can be shared by others.